backend/fleet_management/test_drives.py

```diff
@@ -185,3 +185,9 @@ def test_can_create_a_drive(self):
         )
         self.assertSetEqual({token.is_confirmed for token in tokens}, {False, False})
         self.assertSetEqual({token.is_ok for token in tokens}, {None, None})
+
+    def test_fuel_consumption_is_valid(self):
+        drive = self.drives[0]
```

Comment:
Multiple issues here.

First of all we could and should use factories here. Also if you don't want to use them, I don't really understand why you're usinng `self.drives[0]` and explicitly change it's `start_mileage` and `end_mileage`. That can potentialy make rest of test failing as they can depend on `mileage` values set in `setUp` method and nobody is assuring order of tests execution.

Reply:
@magul As you suggested, I used factories, is it correct right now?
https://github.com/CodeForPoznan/pah-fm/pull/280/commits/f7e2a6e4b0ac5febc7ce711b9e29a7d4705c7909